Doc/library/multiprocessing.rst

```diff
@@ -116,9 +116,9 @@ to start a process.  These *start methods* are
 
   *forkserver*
     When the program starts and selects the *forkserver* start method,
-    a server process is started.  From then on, whenever a new process
+    a server process is spawned.  From then on, whenever a new process
     is needed, the parent process connects to the server and requests
-    that it fork a new process.  The fork server process is single
+    that it forks a new process.  The fork server process is single
```

Comment:
This is an infinitive, _x requests that y do_ is a correct form.
```suggestion
    that it fork a new process.  The fork server process is single
```

Reply:
Thanks, updated.